### PR TITLE
Refactor/restructure validation function

### PR DESCRIPTION
**Pull request by hoomberto**
# Description

Chunked down functions pertaining to form validation and page render

Fixes # (issue)

Fixed the double quote in the no past activities message (Taylor 👀 )

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Open index.html and test

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -143,6 +143,12 @@ function validateIcons() {
   }
 }
 
+function checkInputs(secondsValue, minutesValue, intentionsValue) {
+  (intentionsValue) ? hideError(1) : addError(1);
```
Comment (AshleyOh-bit) [inline main.js:165]: Thank you for including a ternary operator here - it really cleaned up the previous function.

File: main.js [hunk 2]
```diff
@@ -80,34 +80,46 @@ function addToClassList(section, iconId) {
...
+  addToClassList(exerciseSection, "exercise-active");
+  addToClassList(exerciseSection, "active");
+  removeFromClassList(meditateSection, "meditate-active");
+  removeFromClassList(studySection, "study-active")
+  removeFromClassList(meditateSection, "active");
+  removeFromClassList(studySection, "active")
+}
+
 function facilitateIconChange() {
   if (event.target.closest("#study")) {
```
Comment (AshleyOh-bit) [inline main.js:117]: This is MUCH cleaner. Well done.

**Review by AshleyOh-bit — APPROVED** [review]

**Comment by tylrs** [comment]
Looks good, maybe make your description kinder to your teammates

File: main.js (hunk 1 above)
Comment (tylrs) [reply main.js:165]: Yes love the ternaries!

**Review by tylrs — APPROVED** [review]

**Pull request merged**